main/mySpace/admin_view.php

```diff
@@ -9,12 +9,28 @@
 api_block_anonymous_users();
 
 $exportCSV = isset($_GET['export']) && $_GET['export'] === 'csv' ? true : false;
+// Catch param export_csv from buttom
+$exportCSV = isset($_GET['export_csv']) && $exportCSV == false ? true : false;
```

Comment:
This syntax is very confusing... if $exportCSV == false, then return true? Really?